### PR TITLE
[T2A1][T01-T2]Chua Ping Chan

### PR DESCRIPTION
**Pull request by ChuaPingChan**

### CODE REVIEW

**Comment by ChuaPingChan** [comment]
Ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -1142,6 +1142,15 @@ private static String getUsageInfoForExitCommand() {
...
      * ============================
      */
+    
+    /**
+     * Removes prefix from the given fullString if prefix occurs at the start of the string.
+     * 
+     * @return string with prefix removed
+     */
+     private static String removePrefix(String fullString, String prefix) {
+         return fullString.replaceFirst(prefix, "");
```
Comment (nishantbudhdev) [inline src/seedu/addressbook/AddressBook.java]: This method doesn't remove the prefix only.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -31,7 +31,7 @@
  */
 
 /**
- * This class is used to maintain a list of person data which are saved
+ * Maintains a list of person data which are saved
```
Comment (nishantbudhdev) [inline src/seedu/addressbook/AddressBook.java:34]: Good catch. 👍

**Comment by nishantbudhdev** [comment]
@ChuaPingChan Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**